devutils/pyproject.toml

```diff
@@ -28,6 +28,9 @@ tomlkit = "^0.11.8"
 semver = "^3.0.0"
 pylint = "^2.17.4"
 
+[tool.poetry.group.dev.dependencies]
+mdformat = "^0.7.17"
```

Comment:
I'm surprised if this actually fixed things for you. I'm fairly certain this should be in the normal dependencies section to actually get it into use in the downstream package. Poetry _shouldn't_ include dev dependencies from a dependency

Reply:
Just like pylint, black, etc. are currently defined

Reply:
Correct for normal package installations, but storage among others as well will install it in dev mode because of this `robocorp-devutils = { path = "../devutils/", develop = true }`, which attracts the dev ones as well. (and kept this as a dev dep since it's for dev only, when we generate the docs)

Let me know if is best to move it somewhere else.

Reply:
Actually, the best place for `mdformat` should be as a normal dep in [lazydocs](https://github.com/robocorp/lazydocs.git), isn't it?

Reply:
Everything under devutils is development time only, as the name implies. Not a big deal, but all the other tools are are "normal" dependencies.

Actually, what I'm most confused about is why you don't get mdformat. It should come as a dependency for lazydocs (via mdformat-gfm), and I don't need this fix to generate docs for storage.

Reply:
I was getting it, my bad, forgot to run `poetry update` first before running the first `inv docs`.